src/views/account/Homeworks/Atoms/Item.tsx

```diff
@@ -187,7 +187,7 @@ const HomeworkItem = ({ homework, navigation, onDonePressHandler, index, total }
             exiting={FadeOut.duration(200).delay(50)}
           >
             <HTMLView
-              value={`<body>${homework.content}</body>`}
+              value={`<body>${homework.content.replaceAll("<br>", "")}</body>`}
```

Comment:
Peut-être mettre un espace au lieu de rien ?
```suggestion
              value={`<body>${homework.content.replaceAll("<br>", " ")}</body>`}
```

Reply:
ok bonne idée